config/locales/en.yml

```diff
@@ -23,6 +23,497 @@ en:
   hello: "Hello world"
   project_name: "Challenge Platform"
   project_hashtag:  "#ChallengePlatformGenericTag"
+
+  approach_ideas:
+    _header:
+      section_title: "Idea" #line 7
```

Comment:
The line number comments are going to become outdated *very* quickly.  Even now, a handful of these are misaligned.  It's probably not worth the trouble to keep that maintained as better tooling will be able to keep track of where those properties are.  It was probably meant as a temporary thing, which is fine, but it's not tenable for the long haul.